includes/class-newspack-popups-inserter.php

```diff
@@ -127,7 +127,7 @@ public function __construct() {
 		add_filter( 'the_content', [ $this, 'insert_popups_in_content' ], 1 );
 		add_shortcode( 'newspack-popup', [ $this, 'popup_shortcode' ] );
 		add_action( 'after_header', [ $this, 'insert_popups_after_header' ] ); // This is a Newspack theme hook. When used with other themes, popups won't be inserted on archive pages.
-		add_action( 'wp_head', [ $this, 'insert_popups_amp_access' ] );
+		add_action( 'wp_footer', [ $this, 'insert_popups_amp_access' ] );
```

Comment:
In my testing the API request doesn't fire at all, unless I revert to the `wp_head` action. What's the reason for this change?

Reply:
_Accidentally removed the comments by cancelling a review._

I've reverted this change, probably forgot to remove earlier. Works both ways for me, though.

Reply:
Ah no, it's intentional. The change to insert amp-access in the footer hook was to make amp-access aware of any inserted shortcodes: 

https://github.com/Automattic/newspack-popups/blob/8f024171fb4c95d3347397407ab188f16feb372a/includes/class-newspack-popups-inserter.php#L373-L379

This does not cause issues for me with API requests though, I'll investigate further.